From ef21878b962187e16ef31231b857d233af8e9552 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Tue, 23 Aug 2022 16:13:17 +0300
Subject: [PATCH 35/42] ddr: Drop "_mmio" suffix for ddr_lp.c and ddr_utils.c

Dropped the "_mmio" suffix for the ddr_lp.c and ddr_utils.c
files, to align with the naming convention of the files as
generated by DDR Tool.

Issue: ALB-9210
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/nxp/s32/ddr/{ddr_lp_mmio.c => ddr_lp.c}       | 0
 drivers/nxp/s32/ddr/{ddr_utils_mmio.c => ddr_utils.c} | 0
 plat/nxp/s32/s32_common.mk                            | 4 ++--
 plat/nxp/s32/s32g/bl31_sram/bl31_sram.mk              | 4 ++--
 4 files changed, 4 insertions(+), 4 deletions(-)
 rename drivers/nxp/s32/ddr/{ddr_lp_mmio.c => ddr_lp.c} (100%)
 rename drivers/nxp/s32/ddr/{ddr_utils_mmio.c => ddr_utils.c} (100%)

diff --git a/drivers/nxp/s32/ddr/ddr_lp_mmio.c b/drivers/nxp/s32/ddr/ddr_lp.c
similarity index 100%
rename from drivers/nxp/s32/ddr/ddr_lp_mmio.c
rename to drivers/nxp/s32/ddr/ddr_lp.c
diff --git a/drivers/nxp/s32/ddr/ddr_utils_mmio.c b/drivers/nxp/s32/ddr/ddr_utils.c
similarity index 100%
rename from drivers/nxp/s32/ddr/ddr_utils_mmio.c
rename to drivers/nxp/s32/ddr/ddr_utils.c
diff --git a/plat/nxp/s32/s32_common.mk b/plat/nxp/s32/s32_common.mk
index 0287eebef..4ab0d8c3e 100644
--- a/plat/nxp/s32/s32_common.mk
+++ b/plat/nxp/s32/s32_common.mk
@@ -38,8 +38,8 @@ DDR_DRV = ${S32_DRIVERS}/ddr
 DDR_DRV_SRCS += \
 	${DDR_DRV}/ddr_density.c \
 	${DDR_DRV}/ddr_init.c \
-	${DDR_DRV}/ddr_utils_mmio.c \
-	${DDR_DRV}/ddr_lp_mmio.c \
+	${DDR_DRV}/ddr_utils.c \
+	${DDR_DRV}/ddr_lp.c \
 	${DDR_DRV}/ddr_lp_csr.c \
 	${DDR_DRV}/ddrss_cfg.c \
 	${DDR_DRV}/imem_cfg.c \
diff --git a/plat/nxp/s32/s32g/bl31_sram/bl31_sram.mk b/plat/nxp/s32/s32g/bl31_sram/bl31_sram.mk
index eb9b1a10b..86089d178 100644
--- a/plat/nxp/s32/s32g/bl31_sram/bl31_sram.mk
+++ b/plat/nxp/s32/s32g/bl31_sram/bl31_sram.mk
@@ -1,5 +1,5 @@
 #
-# Copyright 2020-2021 NXP
+# Copyright 2020-2022 NXP
 #
 # SPDX-License-Identifier: BSD-3-Clause
 #
@@ -9,7 +9,7 @@ BL31SRAM_SOURCES = plat/common/aarch64/platform_up_stack.S \
 		   plat/nxp/s32/s32g/bl31_sram/bl31sram_main.c \
 		   plat/nxp/s32/s32g/s32g_clocks.c \
 		   plat/nxp/s32/s32g/s32g_mc_me.c \
-		   ${DDR_DRV}/ddr_lp_mmio.c \
+		   ${DDR_DRV}/ddr_lp.c \
 
 BL31SRAM_ARRAY_NAME ?= bl31sram
 BL31SRAM_ARRAY_LEN  ?= bl31sram_len
-- 
2.17.1

